src/backend/access/Makefile

```diff
@@ -8,7 +8,7 @@ subdir = src/backend/access
 top_builddir = ../../..
 include $(top_builddir)/src/Makefile.global
 
-SUBDIRS	    = brin common gin gist hash heap index nbtree rmgrdesc spgist \
+SUBDIRS	    = brin common gin gist hash heap index nbtree pagestore rmgrdesc spgist \
```

Comment:
Why add a pagestore subdirectory for what is essentially a feature we want in core PostgreSQL?

Reply:
I'm not sure that POPS would benefit from this. 
From a community Postgres perspective, I would start it at the access methods (and it suddenly becomes a massive change) with a buffer cache change.